va/va.h

```diff
@@ -980,6 +991,31 @@ typedef union _VAConfigAttribValEncJPEG {
 #define VA_ENC_INTRA_REFRESH_MULTI_REF                  0x00040000
 
 /**@}*/
+/** @name Attribute values for VAConfigAttribHistogram channel. */
+/**@{*/
+typedef enum{
+    VAHistogramChannelNone = 0,
+    VAHistogramChannelY,
+    VAHistogramChannelU,
+    VAHistogramChannelV,
+    VAHistogramChannelR,
+    VAHistogramChannelG,
+    VAHistogramChannelB,
+    VAHistogramChannelA
+}VAHistogramChannelType;
+
+/** \brief Attribute value for VAConfigAttribHistogram */
+typedef union _VAConfigAttribHistogram{
+    struct {
+        /** \brief bin count for given channel. */
+        uint32_t bin_count : 16;
+        /** \brief driver support channel */
+        uint32_t channel   : 8;
+        /** \brief frequence value size */
+        uint32_t reserved  : 8;
+    } bits;
+    uint32_t value;
+} VAConfigAttribHistogramBin;
```

Comment:
It would be better to keep the same name style, _VAConfigAttribHistogram and VAConfigAttribHistogram, or _VAConfigAttribHistogramBin and VAConfigAttribHistogramBin